dasharo-security/secure-boot.robot

```diff
@@ -13,11 +13,16 @@ Resource            ../rtectrl-rest-api/rtectrl.robot
 Resource            ../variables.robot
 Resource            ../keywords.robot
 Resource            ../keys.robot
-
-# TODO:
-# - document which setup/teardown keywords to use and what are they doing
-# - go threough them and make sure they are doing what the name suggest (not
-# exactly the case right now)
+Resource            ../lib/secure-boot-lib.robot
+
+# Required setup keywords:
+# Prepare Test Suite - elementary setup keyword for all tests.
+# Reset Secure Boot Keys For The First Time - resets secure boot keys
+#    for the first time since flashing. Otherwise enabling secure boot
+#    is not possible.
+# Upload Required Images - uploads all required files onto the PiKVM.
```

Comment:
@filipleple  Why is this commented out? How we make sure that required images are on the PiKVM? What happens if we enable it and try to upload every time we run the suite?

Reply:
@macpijan Right, I put it in the Suite Setup/Run Reywords command. 
Thankfully, we can just attempt uploading them every time - the PiKVM handles checking whether they're already there or not on its own.